Added change branch for multiserver zoning shooter tutorial (#1282)
spatialos/UnrealGDK: SpatialGDK/Documentation/content/tutorials/multiserver-shooter/tutorial-multiserver-intro.md

_Image: Example of cross-server shooting using multiserver zoning functionality._ </br></br>

The tutorial demonstrates that the workflows and iteration speed you’re used to as an Unreal developer are almost entirely unaltered by the GDK: it’s just like regular Unreal.

### Change your branch of the Example Project 

> Before starting this tutorial you need to complete the [Example Project set up guide]({{urlRoot}}/content/get-started/example-project/exampleproject-intro). Following this, you clone or download and set up all the elements of the GDK and its dependencies, including the Example Project.

Reviewer: ```suggestion
Before starting this tutorial you need to complete the [Example Project setup guide]({{urlRoot}}/content/get-started/example-project/exampleproject-intro). Following this, you clone or download and set up all the elements of the GDK and its dependencies, including the Example Project.
```
Couple of things:
- set up --> setup
- does "Following this" mean "after this", or "while you follow this"?

To follow the Multiserver zoning shooter tutorial, you need to change your Example Project branch from `release` (which is the default) to `multiserver-tutorial-start`. To do this:

1. If you have Unreal Editor open, close it.
2. In GitHub, change your Example Project branch to `multiserver-tutorial-start`.</br>

> To do this, open a terminal window, navigate to directory containing your cloned Example Project  - this will be    `...\UnrealEngine\samples\UnrealGDKExampleProject` and enter `git checkout -b multiserver-tutorial-start`.

Reviewer: ```suggestion
To do this, open a terminal window, navigate to directory containing your cloned Example Project (this will be    `...\UnrealEngine\samples\UnrealGDKExampleProject`) and enter `git checkout -b multiserver-tutorial-start`.
```

I'm generally a fan of dashes, so was initially going to add a closing dash, but then I thought brackets would probably work better here, for readability.

Reviewer: Also, just thinking, do we need to tell people how to change branch?

I'm thinking that:
- if they're using the GDK, they should know how to change branch already
- if we've included this explanation because we need them to do it this way specifically, we should make that clear, otherwise they'll just do it in the way they're used to (e.g. through the UI).

Author: It's a way to do this not the only way to do this.
I popped it in as hand-hold because it's a tutorial and Unreal is our evaluation platform, so we are likely to get noobs - like CEOs, journos and such-like,  having a go at this and we want to make it as easy as possible for them to see how it works.

3. Open the project in Visual Studio.</br>

> To do this, still in your terminal window, in the directory `...\UnrealEngine\samples\UnrealGDKExampleProject\Game`, double-click on `GDKShooter.sln`.

Reviewer 2: they don't need to be in a terminal window to open up an sln file.

Author: Ta - it's just from Windows Explorer?

> To do this, still in your terminal window, in the directory `...\UnrealEngine\samples\UnrealGDKExampleProject\Game`, double-click on `GDKShooter.sln`.

Author: ```suggestion
To do this, in Windows Explorer, in the directory `...\UnrealEngine\samples\UnrealGDKExampleProject\Game`, double-click on `GDKShooter.sln`.
```

> 4. In Visual Studio, select **Local Windows Debugger** in toolbar to open your project in the Unreal Editor.

Reviewer: ```suggestion
4. In Visual Studio, select **Local Windows Debugger** in the toolbar to open your project in the Unreal Editor.
```

> You can now get going setting up replication of health changes. 

Reviewer: "get going setting up" is a bit of a mouthful - how about "start to set up"?

Author: "You can now set up.."

> You can now get going setting up replication of health changes. 

Author: ```suggestion
You can now set up replication of health changes. 
```
